charts/internal/shoot-system-components/charts/csi-driver-node/templates/clusterrole-csi-resizer.yaml

```diff
@@ -10,6 +10,9 @@ rules:
 - apiGroups: [""]
   resources: ["persistentvolumeclaims"]
   verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
```

Comment:
My understanding is that when `--handle-volume-inuse-error=false` we should not need the RBAC (the watch for Pods).